addon/utils/plugins/table/tab-input-plugin.ts

```diff
@@ -32,55 +33,62 @@ export default class TableTabInputPlugin implements TabInputPlugin {
           return {
             allow: true,
             executor: this.selectFirstCell,
-          }
+          };
         }
       } else if(_manipulation.type === 'moveCursorToEndOfElement' || _manipulation.type === 'moveCursorBeforeElement') {
         if(this.isPreviousElementATable(selection)){
           return {
             allow: true,
             executor: this.selectLastCell,
-          }
+          };
         }
       }
     }
     return null;
   }
 
-  isNextElementATable(selection: ModelSelection) {
+  isNextElementATable = (selection: ModelSelection) => {
```

Comment:
I'm personally not a fan of defining these functions this way. We could move them outside of the class, but defining them as a class field and then binding to this feels odd to me.

Reply:
Not a fan either, I did this in order to solve the typescript error generated 
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/unbound-method.md